### DISCUSSION TITLE
Icons Cheatsheet for the MesloLGM Nerd Font?

### DISCUSSION THREAD

**Discussion by Senbonzakura1234** [JanDeDobbeleer/oh-my-posh#4949] [General]
Is there any icons cheatsheet for the MesloLGM Nerd Font? I can't find it anywhere.
The [cheatsheet i found here](https://www.nerdfonts.com/cheat-sheet) is for nerd font in general, many icons there aren't available in MesloLGM Nerd Font

**Comment by JanDeDobbeleer** [comment]
Validate if your installed font is up to date as that is their only, and correct, cheat sheet.

**Reply by Senbonzakura1234** [reply]
Tks i just figure out that the cheatsheet is correct and include all icons i needs. 

It was vscode terminal's fault, some icons aren't display correctly on its powershell terminal. Normal powershell and the terminal app display the icons fine though, it's only happen with vscode terminal.

@JanDeDobbeleer

**Reply by Senbonzakura1234** [reply]
And it seems only happened on windows 11, i check another system using windows 10, and its vscode display the icons perfectly
@JanDeDobbeleer
